live-examples/js-examples/globalprops-null.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+<code id="static-js">function getVowels(str) {
+  var m = str.match(/[aeiou]/gi);
+  if (m === null) {
```

Comment:
Would you do this, or would you just do `(!m)`? I'm not sure if this matters...

Reply:
I was struggling to find an example that would return null. Some people are in favour of very expressive code so things like `m == false` as opposed to `!m` and I can see that point of view.